### PR TITLE
LPS-176220 | Master

### PR DESCRIPTION
**Pull request by Hanlf**
https://liferay.atlassian.net/browse/LPS-176220

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase [hunk 1]
```diff
@@ -993,6 +993,65 @@ definition {
...
+		ApplicationsMenu.gotoPortlet(
+			category = "System",
+			panel = "Control Panel",
+			portlet = "App Manager");
+
+		AppManager.viewApp(
+			appName = "Knowledge Base",
+			appStatus = "Active");
+
+		AppManager.deactivateApp(appName = "Knowledge Base Web");
```
Comment (austinchiang) [inline modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase:1017]: Can you check if there is a way to exclude the app from deploying or deactivate it from the backend?

File: modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase [hunk 2]
```diff
@@ -1478,4 +1537,30 @@ definition {
 		LexiconCard.viewCardNotPresent(card = "Site A");
 	}
 
+	@description = "The user should see vertial nav items when more than 5 nav items appear on item selector modal."
+	@priority = 4
+	test VertialNavItemsShowsWhenMoreThan5Items {
```
Comment (austinchiang) [inline modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase:1542]: Vertical

File: modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase [hunk 3]
```diff
@@ -1478,4 +1537,30 @@ definition {
 		LexiconCard.viewCardNotPresent(card = "Site A");
 	}
 
+	@description = "The user should see vertial nav items when more than 5 nav items appear on item selector modal."
```
Comment (austinchiang) [inline modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase:1540]: vertical

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please see comments

File: modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase (hunk 1 above)
Comment (Hanlf) [reply modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase]: Use config file to blacklist modules.

File: modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase (hunk 2 above)
Comment (Hanlf) [reply modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase]: updated

File: modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase (hunk 3 above)
Comment (Hanlf) [reply modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase]: updated

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
@austinchiang updated, could you review again?

**Comment by austinchiang** [comment]
Very nice. Thanks @Hanlf !

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**